index.html

```diff
@@ -4,9 +4,16 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>My Recipes</title>
+    
+    </title>
 </head>
+
 <body>
     <h1>My Recipes</h1>
+
+    <li><a hrf="other_recipes/jellof rice.html"></a>jellof rice</li>
+    <li><a hrf="other_recipes/fried rice.html"></a>fried rice</li>
+    <li><a hrf="other_recipes/white rice.html"></a>white rice</li>
```

Comment:
- [ ] What about this `li` parent `ul` or `ol` element?
And you have not fixed the issue I pointed out with your `href`. Is this link clickable from your computer? 

check your code `<a hrf="other_recipes/jellof rice.html"></a>` the `hrf` you hare should be `href`. Please fix this